lib/views/widgets/button.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_ecommerce/utilities/constants/color_manager.dart';
+
+class Button extends StatelessWidget {
+  final String title;
+  final Function onPress;
```

Comment:
Try to make the onPress use the VoidCallBack data type, it will be better to implicitly work with the onpressed parameter.